js/PulsarUIComponent.js

```diff
@@ -1,16 +1,20 @@
-var $             = require('jquery'),
-    dt            = require('datatables.net')(window, $),
-    dt_buttons    = require('datatables.net-buttons')(window, $),
-    dt_responsive = require('datatables.net-responsive')(window, $),
-    dt_select     = require('datatables.net-select')(window, $),
-    countdown     = require('../libs/jquery.countdown/dist/jquery.countdown.min');
+'use strict';
+
+var $ = require('jquery'),
+    $window = window;
```

Comment:
JJ has raised this below, as we get `$($window)` when we jQuerify it, do we need to store a reference to window at all? I think storing a ref to the window as a jQuery object makes sense, but not of the window itself, we get that for free in the global scope.